index.html

```diff
@@ -239,4 +277,6 @@ <h3>
         </div>
       </div>
     </section>
+    <script src="/js/installversion.js" async></script>
+    <script src="/js/osselector.js" async></script>
```

Comment:
I think we should use `defer` instead of `async` as we want the script to be run once the page finished parsing?

Reply:
@solazio these might all be good comments, but I litterally copied the code from the multipass site.  Perhaps ask @anthonydillon and agree changes for both?